src/stores/wordpress-versions-api.ts

```diff
@@ -5,7 +5,8 @@ import { z } from 'zod';
 import { isWordPressDevVersion, isWordPressBetaVersion } from 'src/lib/wordpress-version-utils';
 import { withOfflineCheck } from 'src/stores/utils/with-offline-check';
 
-const MINIMUM_WORDPRESS_VERSION = '5.9.9';
+// Default minimum version for fallback
+const DEFAULT_MINIMUM_WORDPRESS_VERSION = '5.9.9';
```

Comment:
Would it make sense to check all providers here and use the oldest version?

Reply:
You mean the newest?
For example Playground is 6.2.1 and wp-now is 5.9.9, so 6.2.1 would be safe version to use. Did I understand your suggestion correctly?

Reply:
Just updating the min version constant to `6.2.6`?
That sounds good to me, less trouble to maintain both provider constants.

Reply:
@bcotrim I think @sejas meant to remove `DEFAULT_MINIMUM_WORDPRESS_VERSION`, make the version a required param for the query, and always provide the version from the provider.

Reply:
Got it now, thanks!